SIGMA/sigma_build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -x
+set -e
+shopt -s expand_aliases
+
+# Install conda
+export MINICONDA_PYVER="2"
+./bin/conda-inst.sh
+source conda_env.sh
+
+# install deps
+conda install hdf5 autoconf automake libtool
+
+# build and install SIGMA tools
+conda_build cgm
+conda_build moab
+conda_build lasso
+conda_build pytaps
```

Comment:
Right. So if you look at the `pytaps/meta.yaml` file you will see that pytaps only depends on moab.  This simply builds cgm and lasso in case someone wants to use them.